src/components/footer/Footer.component.tsx

```diff
@@ -0,0 +1,23 @@
+import { IconsComponent } from '@/components/icons/Icons.component.tsx';
+
+import footerStyles from './footer.module.css';
+
+export const FooterComponent = () => (
+    <div className={footerStyles.container}>
+        <div className={footerStyles.divider}>
+            <svg xmlns="http://www.w3.org/2000/svg" width="auto" height="2" viewBox="0 0 1148 2" fill="none">
+                <path d="M0 1H1148" stroke="#CCCCCC" />
+            </svg>
+        </div>
+        <>
```

Comment:
you don't need this tag here

Reply:
@serhii-tsivinskyi-moc removed unnecessary tags